src/main/java/org/bookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package org.bookstore.repository;
+
+import java.util.List;
+import org.bookstore.model.Book;
+
+public interface BookRepository {
+    Book save(Book book);
+
+    List<Book> getAll();
```

Comment:
![image](https://github.com/Mariana-VV/BookStoreApp/assets/73041091/ca1c1493-09ea-455d-a239-fae55cd73f4d)
```suggestion
    Book save(Book book);

    List<Book> findAll();
```